packages/labs/context/src/lib/controllers/context-consumer.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {ContextRequestEvent} from '../context-request-event.js';
+import {ContextKey, ContextType} from '../context-key.js';
+import {ReactiveController, ReactiveElement} from 'lit';
+
+export type DispatchOptions = 'connected' | 'updated' | 'raf' | 'microtask';
```

Comment:
For what use cases are these options needed?

Reply:
Good question, I'm struggling to recall the specific uses we had for this. I suspect it was before I came up with the `context-root` solution, when it made sense to have different timings for some consumers where the providers were not yet available.

Reply:
How about we file an issue to consider removing the options, given they may not be needed.

Reply:
I have removed it in 79da874 after discussion with Justin. We can re-add in future if a user needs it.